ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -90,6 +99,13 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
 
     traceLambda(LOG, "blockparam: {}", () -> Json.encodePrettily(blockParam));
 
+    final Optional<List<Withdrawal>> maybeWithdrawals =
+        Optional.ofNullable(blockParam.getWithdrawals())
+            .map(ws -> ws.stream().map(WithdrawalParameter::toWithdrawal).collect(toList()));
+    if (!isWithdrawalsValid(timestampSchedule, blockParam.getTimestamp(), maybeWithdrawals)) {
+      return new JsonRpcErrorResponse(reqId, INVALID_PARAMS);
+    }
+
     if (mergeContext.get().isSyncing()) {
```

Comment:
You can also wire changes to include the withdrawals in the new BlockBody

Reply:
good point, will do